spec/features/admin_claim_tasks_update_with_dqt_api_spec.rb

```diff
@@ -1486,6 +1496,7 @@ def task_outcome
               9,
               1
             ),
+            itt_subjects: ["applied mathematics", "maths and statistics", :maths],
```

Comment:
I don't think we can/would receive a symbol

Reply:
@d-a-v-e thanks, have updated.